curls.sh

```diff
@@ -7,27 +7,27 @@ curl -X POST \
 # Checking payment (if success, you'll get a chatgpt response)
 curl -X POST \
      -H "Content-Type: application/json" \
-     -d '{ "r_hash": "1870fc8372fbfcf42a60838bf070c67bce195d1f928fca4340d8d72f0abb87bc"}' \
+     -d '{"query": "Hi", "user_uuid": "test78e4-a83b-4d8c-b8bd-a0343ca641b4"}' \
```

Comment:
This is definitely needed since we need to associate a query with a user_uuid, but we _also_ need the query that this previously was where we're just checking if the invoice has been paid by it's `r_hash` as the id of the payment. So we need both